UDS-1916: fix gray color alignment with brand

scott-williams-az opened the pull request:
<!-- PREFLIGHT CHECK - to be deleted before submitting PR -->
<!--   - CHECK: Do tests need to be added or updated? -->
<!--   - CHECK: Are data layer additions or updates needed? -->

### Description

<!-- Description of problem -->
<!-- Solution -->
<!-- Testing Steps -->

## Checklist

- [ ] Tests pass for relevant code changes

## Important Reminders
<!-- Add meaningful tests -->
<!-- Remove tests that do not provide value -->

### Links

- [JIRA ticket](https://asudev.jira.com/browse/UDS-1916)
- [Unity reference site](https://asu.github.io/asu-unity-stack/)
- [Unity Design Kit](https://shared-assets.adobe.com/link/fb14b288-bf63-47e0-5d30-384de5560455)

oatkar committed:
fix(unity-bootstrap-theme): blockquotes in mobile view too compressed, need to stack

scott-williams-az committed:
feat(app-degree-pages): allow list view including both  GR and UGCM certs

scott-williams-az committed:
remove comment

oatkar committed:
fix(unity-bootstrap-theme): cSS Padding added to images without header

davidornelas11 committed:
chore: update eslint to v9 and replace eslint config files

davidornelas11 committed:
chore: remove components-core as it's deprecated

davidornelas11 committed:
chore: remove shared folder from root

davidornelas11 committed:
feat: add shared to its own monorepo package

davidornelas11 committed:
chore: run new eslint fixes and update to "@asu/shared" package on all packages

Gamille7 committed:
chore: update gtm documentation to remove ET Enterprise container references now that we've merged the ET Enterprise Container and Enterprise Hub Container.

Gamille7 committed:
UDS-1941: Cookie Consent Opt Out in Footer

Gamille7 committed:
fix: button type for cookie consent opt out is now a button

davidornelas11 committed:
fix(component-header-footer): update url params for a11y to use fragments instead of params

Gamille7 committed:
fix: styles for manual consent opt out

Gamille7 committed:
fix: styles for manual consent opt out version 2

davidornelas11 committed:
fix(component-header-footer): fix school logo in footer styles

davidornelas11 committed:
fix(unity-bootstrap-theme): fix school image styles in global footer

davidornelas11 committed:
chore(unity-react-core): update img imports in stories to use imageAny func

davidornelas11 committed:
build(shared): add correct dependencies

davidornelas11 committed:
fix(app-webdir-ui): add @asu/shared to dependencies

davidornelas11 committed:
chore(shared): add linting to shared package

scott-williams-az committed:
fix(unity-bootstrap-theme): fix gray color alignment with brand

brand gray scale has 1-7 (dark to light), This change aligns css to follow 1-dark to 7-light

scott-williams-az committed:
fix(unity-bootstrap-theme): fix gray color alignment with brand

brand gray scale has 1-7 (dark to light), This change aligns css to follow 1-dark to 7-light

scott-williams-az committed:
fix(unity-react-core): fix gray color alignment with brand

brand gray scale has 1-7 (dark to light), This change aligns css to follow 1-dark to 7-light

asu-jenkins-devops commented:
Storybook deployed at https://unity-uds-staging.s3.us-west-2.amazonaws.com/pr-1557/index.html

Gamille7 committed:
fix(component-header-footer): updating footer to included privacy button

semantic-release-bot committed:
chore(release): 1.0.11 [skip ci]

# [@asu/component-header-footer-v1.0.11](https://github.com/asu/asu-unity-stack/compare/@asu/component-header-footer-v1.0.10...@asu/component-header-footer-v1.0.11) (2025-07-11)

### Bug Fixes

* button type for cookie consent opt out is now a button ([f7cff20](https://github.com/asu/asu-unity-stack/commit/f7cff2084614f88213535d5c3f186d508c5eff1d))
* **component-header-footer:** updating footer to included privacy button ([fd022c4](https://github.com/asu/asu-unity-stack/commit/fd022c45ca846c1fa1e2213e951fa431e2b483a6))
* styles for manual consent opt out ([b028c9f](https://github.com/asu/asu-unity-stack/commit/b028c9fb6ba32aeaa6dd1dbfb1cdf572e2356740))
* styles for manual consent opt out version 2 ([35f4452](https://github.com/asu/asu-unity-stack/commit/35f4452c3c32ff13c71dbbac5c9386774b3bf001))

semantic-release-bot committed:
chore(release): 1.26.3 [skip ci]

# [@asu/unity-bootstrap-theme-v1.26.3](https://github.com/ASU/asu-unity-stack/compare/@asu/unity-bootstrap-theme-v1.26.2...@asu/unity-bootstrap-theme-v1.26.3) (2025-07-11)

### Bug Fixes

* **component-header-footer:** updating footer to included privacy button ([fd022c4](https://github.com/ASU/asu-unity-stack/commit/fd022c45ca846c1fa1e2213e951fa431e2b483a6))
* styles for manual consent opt out ([b028c9f](https://github.com/ASU/asu-unity-stack/commit/b028c9fb6ba32aeaa6dd1dbfb1cdf572e2356740))
* styles for manual consent opt out version 2 ([35f4452](https://github.com/ASU/asu-unity-stack/commit/35f4452c3c32ff13c71dbbac5c9386774b3bf001))
* **unity-bootstrap-theme:** blockquotes in mobile view too compressed, need to stack ([84cb73f](https://github.com/ASU/asu-unity-stack/commit/84cb73f260ded56e088555da91debc999f872889))

semantic-release-bot committed:
chore(release): 1.26.4 [skip ci]

# [@asu/unity-bootstrap-theme-v1.26.4](https://github.com/ASU/asu-unity-stack/compare/@asu/unity-bootstrap-theme-v1.26.3...@asu/unity-bootstrap-theme-v1.26.4) (2025-07-14)

### Bug Fixes

* **unity-bootstrap-theme:** cSS Padding added to images without header ([5d3a4f2](https://github.com/ASU/asu-unity-stack/commit/5d3a4f2170346ca5f5b4a8a17fcbcb1c9a4f863c))

davidornelas11 committed:
chore: update lock file

semantic-release-bot committed:
chore(release): 1.0.12 [skip ci]

# [@asu/component-header-footer-v1.0.12](https://github.com/asu/asu-unity-stack/compare/@asu/component-header-footer-v1.0.11...@asu/component-header-footer-v1.0.12) (2025-07-14)

### Bug Fixes

* **component-header-footer:** fix school logo in footer styles ([f25e7c1](https://github.com/asu/asu-unity-stack/commit/f25e7c1fce0ead0f3f2be0d50cd3faeb91d82f8e))
* **component-header-footer:** update url params for a11y to use fragments instead of params ([9f0481a](https://github.com/asu/asu-unity-stack/commit/9f0481ae63b465dea65452b1e93bd58596d5ac87))

semantic-release-bot committed:
chore(release): 1.26.5 [skip ci]

# [@asu/unity-bootstrap-theme-v1.26.5](https://github.com/ASU/asu-unity-stack/compare/@asu/unity-bootstrap-theme-v1.26.4...@asu/unity-bootstrap-theme-v1.26.5) (2025-07-14)

### Bug Fixes

* **unity-bootstrap-theme:** fix school image styles in global footer ([e31f753](https://github.com/ASU/asu-unity-stack/commit/e31f75336f71a2f51a5e03d1d13069c38891688b))

semantic-release-bot committed:
chore(release): 3.1.0 [skip ci]

# [@asu/app-degree-pages-v3.1.0](https://github.com/asu/asu-unity-stack/compare/@asu/app-degree-pages-v3.0.5...@asu/app-degree-pages-v3.1.0) (2025-07-14)

### Features

* **app-degree-pages:** allow list view including both  GR and UGCM certs ([8dd70c4](https://github.com/asu/asu-unity-stack/commit/8dd70c41c0993d3e9f5d4c1c8bb518a2086e944f))

davidornelas11 committed:
style: linting ran again

davidornelas11 committed:
build: add shared packaged to ignore publish step

scott-williams-az committed:
Merge branch 'dev' into UDS-1916

scott-williams-az committed:
refactor(component-header-footer): update css var names to align with brand definition of gray 1-7

scott-williams-az committed:
chore: fix various lint issues

scott-williams-az commented:
replaced by https://github.com/ASU/asu-unity-stack/pull/1563
